src/plugins/discover/public/components/discover_tour/discover_tour_provider.tsx

```diff
@@ -215,13 +211,13 @@ export const DiscoverTourProvider = ({
   );
   const tourSteps = useMemo(
     () =>
-      isPlainRecord
```

Comment:
@davismcphee it so nice to see this go away ... the naming wasn't good anyway, I wonder who ... ah .. it was me :) 
I wonder why I didn't I name this isBoringRecord

Reply:
What is it about? What is the mentioned functionality?

Reply:
@stratoula It's a tour explaining how to use the data grid. For data view mode it has more steps configured. 

I can address that separately to show all steps for ES|QL mode too, since we've added more features to it.

Reply:
Sounds fantastic, thanx Julia!